client/src/components/Details.js

```diff
@@ -0,0 +1,63 @@
+import "../components/Details.css";
+import { useState, useEffect } from "react";
+import { useParams, useHistory } from "react-router-dom";
+
+
+function Carousel() {
+  
+}
+
+export default function Details() {
+  const { ID } = useParams();
+  const history = useHistory();
+  const [property, setProperty] = useState({});
+ 
+    console.log("Hallo von Details")
+    console.log(ID)
+  useEffect(() => {
+      const url = `http://localhost:4000/properties/${ID}`;
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((apiData) => {
+        setProperty(apiData[0]);
+        console.log(apiData);
+      });
+  }, [ID]);
+
+
+  return (
+    <div className="detailDiv">
+      <div className="p-1">
+        <div className="p-1-1">
+          <p className="p-a">{property.City}</p>
+          <p className="p-a">{property.Street}</p>
+        </div>
+        <div className="p-1-2">
+          <p className="p-a">{property.Price}</p>
+          <p className="p-a">ID: {property.ID}</p>
+        </div>
+      </div>
+      <hr className="hr"></hr>
+      <div className="p-1">
+        <div className="p-2-1">
+          <p className="p-m1">Year Build</p>
+          <p className="p-m1">Interior</p>
+          <p className="p-m1">Bedrooms</p>
+          <p className="p-m1">Status</p>
+        </div>
+        <div className="p-2-2">
+          <p className="p-m2">{property.YearBuild}</p>
+          <p className="p-m2">{property.Interior}</p>
+          <p className="p-m2">{property.Bedrooms}</p>
+          <p className="p-m2">{property.Status}</p>
+        </div>
+      </div>
+      <hr className="hr"></hr>
+      <p className="description">{property.Discription}</p>
+      <button className="btn" onClick={() => history.goBack()}>
+        Back
+      </button>
+    </div>
```

Comment:
Aren't there more semantic elements than `<div>`, which you could use here? Also, the class names and ruleset declarations they represent scream for a layout system like `flex` or `grid`

Reply:
used flex. I think I would not need all the divs with grid